### ISSUE TITLE
Error code gets propagated incorrectly

### ISSUE THREAD

**Issue by PerilousApricot** [dmwm/CRABServer#4679]
A few times CMSWEB will be temporarily down and the following chain of events will happen. Though the wrapper is trying to set an exit code of 80000, it shows up as 128 everywhere (monitoring, etc)

http://glidemon.web.cern.ch/glidemon/show.php?log=http://vocms096.cern.ch/mon/cms931/150222_183020_crab3test-3:meloam_crab_ra2tau_skim2015_v3_QCD_Pt_20_MuEnrichedPt_15_TuneZ2star_8TeV_pythia6_Summer12_DR53X-PU_S10_START53_V7A-v1_testing/job_out.425.0.txt

Job 425 from task 150222_183020_crab3test-3:meloam_crab_ra2tau_skim2015_v3_QCD_Pt_20_MuEnrichedPt_15_TuneZ2star_8TeV_pythia6_Summer12_DR53X-PU_S10_START53_V7A-v1_testing

**Comment by PerilousApricot** [comment]
Also, CMSWEB being down should probably be handled as a recoverable error.

**Comment by belforte** [comment]
but is it understood how CMSWEB being down can cause the exit code to be propagated incorrectly ?

**Comment by PerilousApricot** [comment]
I think it's just not properly being trapped upstream

**Comment by mmascher** [comment]
Hi, I think it could be related to https://github.com/dmwm/CRABServer/issues/4642
We are reporting the short exit code instead of the extended one, indeed:

```
>>> 80000%256
128
```

**Comment by mmascher** [comment]
I fixed #4642 (I will merge the pull soon after we are done with the March release)

I am reporting the long exit code to the dashboard. The crab status was already using it. The short exit code is still used in glidemon (we need Dima), and some places in this script (*) (and the joblog) where I specified we are printing the short exit code.

About the CMSWEB unavailabiliity, that particular error need to be solved by removing the CMSWEB phedex dependency in the jobwrapper (need to take the SE PNN from siteconfig).

If there is no objection I would like to close this one.

(*) https://github.com/dmwm/CRABServer/blob/master/scripts/gWMS-CMSRunAnalysis.sh
